tests/Unit/PreservableParamsTest.php

```diff
@@ -85,6 +85,7 @@ public function data_get_preservable_querystring_params_from_url() {
 
 	/**
 	* @covers \Automattic\LegacyRedirector\Lookup::get_preservable_querystring_params_from_url
+	* @covers \Automattic\LegacyRedirector\Utils::mb_parse_url
```

Comment:
Why does would the test for preservable querystring params be covering the code from the Utils method?

I think you want `@uses` here.
```suggestion
	* @uses \Automattic\LegacyRedirector\Utils::mb_parse_url
```

That acknowledges that `Utils::mb_parse_url()` gets called as part of the test, but it's not the subject under test.